Proof-of-Stake.md

```diff
@@ -118,10 +118,10 @@ Aside from costing a massive amount of money, the consensus mechanism does have
 
 ### The Downsides
 
-Not all is rainbows and unicorns in this world however, and with it's many advantages, PoS does have a few cons that continue to make consensus mechanisms an active field of research and innovation.
+With its many advantages, PoS does have a few cons that continue to make consensus mechanisms an active field of research and innovation.
 
-1. Proof of Stake is younger and less battle-tested than Proof of Work, which has stood the test of time ever since Bitcoin was created till date.
-2. Proof of Stake protocols are much more complex to implement in code, technically, which also increases the surface area for possible bugs and issues to arise.
+1. Proof of Stake is younger and less battle-tested than Proof of Work, which has stood the test of time ever since Bitcoin was created.
+2. Proof of Stake protocols are much more complex to implement in code, technically, which also increases the surface area for possible bugs and issues.
```

Comment:
I think this was good as it was @SlavaTimbaliuc

Reply:
well "till" isn't a read word lol so I think that needs some rewording

Reply:
Alright, well in that case, the first sentence can be changed and then the second sentence can be left as is.

Reply:
First sentence changed, second sentence the same